docs/source/conf.py

```diff
@@ -77,34 +77,27 @@
 
 # The code below will cut version displayed in the dropdown like this:
 # tags like v0.1.0 = > 0.1
-# branch like release/0.1 => 0.1
 # main will remain main
 # if not set will fail back to main
 # the version varible is used in layout.html: https://github.com/pytorch/executorch/blob/main/docs/source/_templates/layout.html#L29
 if et_version_docs:
-    # Check if starts with release/ and set the version to the number after slash
-    if et_version_docs.startswith("release/"):
+    if et_version_docs.startswith("refs/tags/v"):
+        version = ".".join(
+            et_version_docs.split("/")[-1].split("-")[0].lstrip("v").split(".")[:2]
+        )
+        print(f"Version: {version}")
+        release = version
+        html_title = " ".join((project, version, "documentation"))
+    elif et_version_docs.startswith("refs/heads/"):
         version = et_version_docs.split("/")[-1]
```

Comment:
We have a lot of heads with `/`-separated components. Does it always make sense to use the final component? Or should this use everything after `refs/heads/`, maybe converting slashes to something else?

Please add a comment describing the general intent of this, and what we expect the `version` would be, with some examples.